src/auto_archiver/enrichers/wayback_enricher.py

```diff
@@ -70,11 +71,16 @@ def enrich(self, to_enrich: Metadata) -> bool:
             return False
 
         # check job status
-        job_id = r.json().get('job_id')
-        if not job_id:
-            logger.error(f"Wayback failed with {r.json()}")
+        try:
+            job_id = r.json().get('job_id')
+            if not job_id:
+                logger.error(f"Wayback failed with {r.json()}")
+                return False
+        except json.decoder.JSONDecodeError as e:
+            logger.error(f"Expected a JSON with job_id from Wayback and got {r.text}")
```

Comment:
Is it worth adding a length restriction to the text or streaming in case of large responses, or does it stay within a predictable length?